src/store/metadata/stub/content/datasets.json

```diff
@@ -267,8 +267,53 @@
               "modified": "2022-08-18T10:51:00Z"
             }
           ]
+      },
+      {
+        "@id": "https://staging.idpd.uk/datasets/DLUHC",
+        "@type": "dcat:DatasetSeries",
+        "identifier": "existing_dwellings_data",
+        "title": "domestic Energy Performance Certificates for existing dwellings",
+        "summary": "Data from certificates for existing domestic properties lodged on the Energy Performance of Buildings Registers, by average energy efficiency rating",
+        "description": "Data from certificates for existing domestic properties lodged on the Energy Performance of Buildings Registers, by average energy efficiency rating",
+        "issued": "2023-07-27T09:30:00Z",
+        "modified": "2023-07-27T09:30:00Z",
+        "next_release": "2024-01-26T09:30:00Z",
+        "publisher": "https://staging.idpd.uk/publishers/office-for-national-statistics",
+        "creator": "https://staging.idpd.uk/publishers/office-for-national-statistics",
+        "contact_point": {
```

Comment:
JSON file needs to be formatted to have proper nesting

Reply:
Proper formatting is done